build.gradle

```diff
@@ -65,9 +65,9 @@ allprojects {
     targetCompatibility = 1.8
 
     project.ext.versions = [
-            kafka             : '2.1.1',
+            kafka             : '2.7.0',
```

Comment:
Hi, thank you very much for this PR. We will need to be careful with it as you updated many dependencies. Especially we are sensitive about Kafka libraries update as the last one introduced problems in our environments. More information about it you can find in the [changelog](https://github.com/allegro/hermes/blob/master/CHANGELOG.md#1276-downgraded-kafka-to-211). I think that in the first place we will test this PR very well as SNAPSHOT on one of our environments and merge it only when we consider it as safe. I will also consult this with my team as probably safer is to introduce Kafka libraries update in a separate PR. Thanks again!